apm_client/generate_protos.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
is this file only for python, or for all languages ?

Reply:
Added some documentation to the script!